src/components/configuration.rs

```diff
@@ -55,6 +55,7 @@ pub struct Config {
     pub wkc_metrics_bearer_token: String,
     pub redis: RedisConfig,
     pub cache_hashing_key: String,
+    pub friends_stream_page_size: String,
```

Comment:
I think it would be best to change this to the appropriate type here (e.g. an integer in this case) to be able to use it downstream with no need to transform it.

Reply:
definitely!